app/views/kudos/index.html.erb

```diff
@@ -20,3 +21,23 @@
     <% end %>
   <% end %>
 </div>
+<% end %>
+<% if @kudos %>
+<!--Descriptive page name, messages and instructions-->
+<h2 class="heading"><%= ts('Kudos History') %></h2>
+<!--/descriptions-->
+<!--main content-->
+<% if logged_in? && @kudos.present? %>
```

Comment:
If they haven't left any kudos, is it worth an else statement with a helpful message to explain, "You haven't left any kudos yet. Once you've given kudos to some works, your history will show here."